src/Fixer/Phpdoc/PhpdocOrderFixer.php

```diff
@@ -65,11 +65,6 @@ public function getDefinition()
      */
     public function getPriority()
     {
-        /*
-         * Should be run before the php_doc_separation fixer so that if we
-         * create incorrect annotation grouping while moving the annotations
-         * about, we're still ok.
```

Comment:
lets keep for now, especially since we've no test that shows this

Reply:
> we've no test that shows this

How come? It's in the doc comment above, isn't it?